### PR TITLE
Track torch.compile calls

### PR DESCRIPTION
**Pull request by williamwen42**
Title.

### CODE REVIEW

File: torch/__init__.py [hunk 1]
```diff
@@ -1147,6 +1147,8 @@ def foo(x):
             return torch.sin(x) + torch.cos(x)
 
     """
+    from . import _C
+    _C._log_api_usage_once("torch.compile")
```
Comment (colin2328) [inline torch/__init__.py:1151]: LGTM. cc @Chillee  @jansel if you ahve conerns

**Review by colin2328 — APPROVED** [review]

File: torch/__init__.py (hunk 1 above)
Comment (williamwen42) [reply torch/__init__.py:1151]: the `_C` in front of `_log_api_usage_once` is still needed, but `_C` is already imported, so I can remove the import statement.

**Review by anijain2305 — APPROVED** [review]

**Pull request closed without merging**